disnake/interactions/application_command.py

```diff
@@ -174,16 +174,19 @@ class ApplicationCommandInteractionData(Dict[str, Any]):
         All resolved objects related to this interaction.
     options: List[:class:`ApplicationCommandInteractionDataOption`]
         A list of options from the API.
+    guild_id: Optional[:class:`int`]
+        ID of the guild the command is registered to.
```

Comment:
```suggestion
        ID of the guild the command is being called from.
```

Currently it implies to me at-least that this command can only be registered in a single guild and is a bit ambigeous in the context of this is an interaction

Reply:
Your implication is correct, because in this case by "command" we mean an API command (with a unique ID, version, guild ID, etc.) so it indeed exists only in one guild